### PR TITLE
1.3.2 Release - website changes

### PR DESCRIPTION
**Pull request by krisfreedain**
Signed-off-by: Kris Freedain <freedak@amazon.com>

### Description
Website updates to support the 1.3.2 release
 
### Issues Resolved
N/A

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _artifacts/opensearch-1.3/arm64/opensearch-1.3.2-linux-arm64-rpm.markdown [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+role: daemon
+artifact_id: opensearch
+version: 2.0.0-rc1
+platform: linux
+architecture: arm64
+artifact_url: https://artifacts.opensearch.org/releases/bundle/opensearch/2.0.0-rc1/opensearch-2.0.0-rc1-linux-arm64.rpm
```
Comment (zelinh) [inline _artifacts/opensearch-1.3/arm64/opensearch-1.3.2-linux-arm64-rpm.markdown:7]: The version and URL for this file will need to be changed. 
https://artifacts.opensearch.org/releases/bundle/opensearch/1.3.2/opensearch-1.3.2-linux-x64.rpm

File: _artifacts/opensearch-1.3/arm64/opensearch-1.3.2-linux-arm64-yum.markdown [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+role: daemon
+artifact_id: opensearch
+version: 2.0.0-rc1
+platform: linux
+architecture: arm64
+slug: opensearch-2.0.0-rc1-linux-arm64-yum
+category: opensearch
+type: yum
+guide: https://opensearch.org/docs/latest/opensearch/install/rpm
+artifact_url: https://artifacts.opensearch.org/releases/bundle/opensearch/2.x/opensearch-2.x.repo
```
Comment (zelinh) [inline _artifacts/opensearch-1.3/arm64/opensearch-1.3.2-linux-arm64-yum.markdown:11]: The version and URL for this file will need to be changed.

File: _artifacts/opensearch-1.3/arm64/opensearch-1.3.2-linux-arm64-rpm.markdown (hunk 1 above)
Comment (krisfreedain) [reply _artifacts/opensearch-1.3/arm64/opensearch-1.3.2-linux-arm64-rpm.markdown]: fixed

File: _artifacts/opensearch-1.3/arm64/opensearch-1.3.2-linux-arm64-yum.markdown (hunk 2 above)
Comment (krisfreedain) [reply _artifacts/opensearch-1.3/arm64/opensearch-1.3.2-linux-arm64-yum.markdown]: fixed

File: opensearch-2.0.0-rc1-linux-arm64-rpm.markdown [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (zelinh) [inline opensearch-2.0.0-rc1-linux-arm64-rpm.markdown:1]: Is this file needed?

**Review by bbarani — APPROVED** [review]
